### ISSUE TITLE
pScheduler ModuleNotFound

### ISSUE THREAD

**Issue by andrewtakeshi** [perfsonar/pscheduler#1177]
I'm having issues using pScheduler through python. Specifically, I'm trying to use the traceroute tool to run traceroutes remotely as part of a network exploration tool I'm building. pScheduler works to run traceroutes both remotely and locally through the command line, but it fails to work when I use `subprocess` in Python to do the same thing. From the command line I'm not getting errors when I run 
`pscheduler troubleshoot`. 

I'm on Centos 7.9.2009, kernel version 3.10.0-1127.19.1.el7.x86_64, and I'm using perfsonar-toolkit 4.4.0-1.el7. I'm not sure if this is relevant, but I have both python-pscheduler.noarch (4.2.4-1.el7) and python3-pscheduler.noarch (4.4.0-1.el7) installed. I'm also using Python 3.6.8.

I'm trying to use this in python:
```python
subprocess.Popen(['pscheduler', 'task', 'trace', '-s', source, '-d', dest], stdout=subprocess.PIPE, universal_newlines=True)
```
When I run this, I get the error:
```python
Traceback (most recent call last):
    File "/usr/libexec/pscheduler/commands/task", line 10, in <module>
        import pscheduler
ModuleNotFoundError: No module named 'pscheduler'
```

This was working previously; I'm not sure what version of the perfsonar-toolkit I was using, but updating to 4.4.0-1 seems to have broken this. I'm also very confused, as afaik `subprocess` should be nearly identical to opening a new shell and running the command there, which works just fine. 

I looked at #1130, but given that I'm able to successfully run `pscheduler troubleshoot` I think it may be a separate issue. 

Any help would be greatly appreciated!

**Comment by andrewtakeshi** [comment]
I compared the path between the venv I was using and the base interpreter, and there were some things missing. After adding them it seems to be working, sorry about that.

**Comment by mfeit-internet2** [comment]
Instead of invoking the CLI, which is for humans,  there are a couple of better ways to get pScheduler to do your bidding programmatically:

First (preferred) is to use the [batch processor](https://docs.perfsonar.net/pscheduler_batch.html), which will take care of all of the interactions with pScheduler, lets you run multiple measurements serially or in parallel and returns the results as easy-to-handle data.  On a system which has the `python3-pscheduler` module installed, It can be [invoked directly from Python](https://docs.perfsonar.net/pscheduler_batch.html#invocation-from-python).

Second (less-preferred) is to [use the API directly](https://github.com/perfsonar/pscheduler/blob/master/pscheduler-docs/pscheduler-docs/api/api-example).
